uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrixTest.php

```diff
@@ -73,28 +73,101 @@ public function testCanTransitionToForLeaveApproverReturnsFalseForAllNonPossible
   }
 
   public function testCanTransitionToForAdminForReturnsTrueAllPossibleTransitionStatuses() {
+    $adminID = 5;
```

Comment:
They are just random contact ID's. Normally the ID is supposed to be gotten from the contact fabricator but since Its not really needed for the contact to exist in DB before registering it in the session. This allows the test run time to be significantly faster.